lib/netsuite/records/serialized_inventory_item_locations.rb

```diff
@@ -0,0 +1,41 @@
+module NetSuite
+  module Records
+    class SerializedInventoryItemLocations
```

Comment:
@sherriously This doesn't look like it's a valid NetSuite record. Are you trying to implement a list here? lists should use the `Support::Sublist` base class.